itou/eligibility/models/iae.py

```diff
@@ -108,6 +108,28 @@ def last_expired(self, job_seeker, for_siae=None):
 
         return last
 
+    def last_for_job_seeker(self, job_seeker, prefetch=None):
```

Comment:
```suggestion
    def last_for_job_seeker(self, job_seeker):
```

À retirer si l'on ne souhaite pas paramétriser les prefetch